tests/elements-regression/tests/elements-regression.test.js

```diff
@@ -77,9 +85,5 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 			await expect( page.locator( EditorSelectors.container ) ).toHaveScreenshot( `${ widgetType }_published.png`, { maxDiffPixels: 100 }, { timeout: 10000 } );
 		} );
 	}
-
-	test.afterAll( async ( { request } ) => {
```

Comment:
Why don't we want to clean up after ourselves?

Reply:
sometimes for some weird reasons, afterAll hook deletes all the images before the test with images is passed. This causes flakiness. Some images are not loaded, because they are deleted.
There should be no impact for ci because the docker image will be deleted.

Reply:
https://playwright.dev/docs/api/class-test#test-before-all. So it does before all, after all for each worker, which means we are adding media and deleting 3 times in parallel which of course brings flakiness. 
The solution is to move the upload and delete of media to global-setup.js